extra/1-radio-stations.js

```diff
@@ -15,6 +15,14 @@
 
 // `getAllFrequencies` goes here
 
+function getAllFrequencies() {
+  let array = [];
```

Comment:
Very good.
A very minor point: you can actually declare array as `const`. Even if you add elements to the list using `push`, you're not changing the value of `array` which is still the same array even when its content changes.
```
const array = [];
array.push('hello'); // correct
array = ['hello']; // error because you're assigning a new array to 'array'
```
Don't worry if it seems a bit obscure. It is!